content/momentum/changelog/4/4-5-0.md

```diff
@@ -0,0 +1,79 @@
+---
+lastUpdated: "11/10/2022"
+title: "Momentum 4.5.0 Changelog"
+description: "Momentum 4.5.0 was released on 2022-12-01. This section will list all of the major changes that happened with the release of Momentum 4.5.0 Depending on installation type, all changes may not be applicable"
+---
+
+**Momentum 4.5.0** was released on 2022-12-01. This section will list all of the major changes that happened with the release of **Momentum 4.5.0**. Depending on installation type, all changes may not be applicable
+
+<a name="changelog.4.5.0.table"></a> 
+
+| Type | Ticket | Description |
+| --- | --- | --- |
+|SD-2813|Story|Only configure a single mta_id on instances
+|SD-2863|Story|Fix High "Overflow in uncontrolled allocation size"
+|SD-2864|Story|Fix High "Time-of-check time-of-use filesystem race condition"
+|SD-2866|Story|Fix High "Not enough memory allocated for array of pointer type" alert
+|SD-2871|Story|Fix Error "Inconsistent nullness check"
+|SD-2873|Story|Fix Error " Dead code due to goto or break statement" alert
+|SD-2875|Story|Security Exception Warning "Comparison result is always the same" alert
+|SD-2876|Story|Fix Warning "Ambiguously signed bit-field member" alert
+|SD-3354|Story|ET servers running exploitable nginx in MTA2
+|SD-3615|Story|Improve templating language documentation
+|SD-3619|Story|Fix crashes in Sieve when creating new messages
+|SD-2302|Bug|bounce address localpart HMAC validation is not case insensitive
+|SD-2935|Bug|Crash when outbound_throttle_messages is configured at the same time as core_mailq_throttle_message hook
+|SD-2995|Bug|soft_bounce functionality periodically tries using an invalid connection handle causing a crash
+|SD-3167|Bug|Stack overflow in modules/delivery/http2/http2clnt.c http_send
+|SD-3259|Bug|Momentum incorrectly interprets checkpoint file data on restart
+|SD-3263|Bug|DKIM Oversigning to Avoid Replay Attacks
+|SD-3288|Bug|Transmission API does not sanity check recipient list for empty array
+|SD-3335|Bug|lu_pull not working on AL2 nodes
+|SD-3374|Bug|lu_pull not working on AL2 nodes, round 2
+|SD-3411|Bug|gstack_ecelerity quiet mode does not work on AL2
+|SD-3497|Bug|DSNs are not Generated when RCPTO is Upper Case
+|SD-3535|Bug|extract_sender_from_msg moved to msys namespace
```

Comment:
We need a ticket to add this to the main documentation for Lua policy

Reply:
No customer has ever asked for this functionality.  Not worth documenting.